usaspending_api/disaster/v2/views/overview.py

```diff
@@ -57,6 +58,18 @@ def funding(self):
             .values("def_code", "amount")
         )
 
+    def unobligated_balance(self):
```

Comment:
Is there a reason `budget_authority_unobligated_balance_brought_forward_cpe` couldn't be included in the original funding ORM query? Seems like the annotation could include a sum for both fields

Reply:
I have combined these queries into one for clarity and performance.